### PR TITLE
New player guidance

### PR DESCRIPTION
**Pull request by sigurdfdragon**
Posting for comments on suitability, especially for tutorial change.

This goes along with beginner campaign reordering in a273a8df25f1721b0029e83d9d2b17dfb3127f11

Will merge in about a week if no objections.

### CODE REVIEW

**Comment by gfgtdf** [comment]
why do we give tht hint only on hard? wouldn't it make more sense to give more hints on lower difficulcies?


That said i really don't know about the current tutorials state. I _vaguely_ remember the first time i played wesnoth and started with the tutorial and understood nothing. I had no idea i shodul do or what "turn based strategy" is. But that was quite some time ago (1.6 i think it was) so it might have changed since then.

File: data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg [hunk 1]
```diff
@@ -172,6 +172,11 @@ Will he heed the call? I do not know if he has kept the amulet; we have not spok
                 bonus=yes
                 carryover_percentage=40
             [/gold_carryover]
+#ifdef HARD
```
Comment (CelticMinstrel) [inline data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg]: I think you meant `#ifndef HARD`; though, I think `#ifdef EASY` might be more appropriate.

**Comment by sigurdfdragon** [comment]
I should probably add a code comment, one has to look to see that the skeletons are only on hard difficulty in AToTB S1.

**Comment by CelticMinstrel** [comment]
In that case I kinda wonder whether a hint is needed at all. Would a beginner be playing on hard?

**Comment by sigurdfdragon** [comment]
Not sure, but I could see some players jumping into hard right after the tutorial, which is why I thought it might be useful

**Comment by pydsigner** [comment]
The convention throughout the Wesnoth beginner campaigns is to not show
extra help on the hard difficulties.

File: data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg [hunk 2]
```diff
@@ -1063,7 +1063,7 @@ A full list of abilities and weapons specials, along with traits, may be found i
 
         [message]
             speaker=Galdrad
-            message= _ "You have beaten the orcs! You may want to try some novice-level campaigns next, such as: <i>The South Guard</i>, <i>An Orcish Incursion</i>, <i>A Tale of Two Brothers</i>, or <i>Heir to the Throne</i>. <i>The South Guard</i> was specifically designed as a beginner’s campaign. Konrad, Li’sar and Delfador are characters from <i>Heir to the Throne</i>. You can also refer to the in-game help browser if you ever need to refresh your memory on gameplay mechanics."
+            message= _ "You have beaten the orcs! You may want to try one of the novice-level campaigns next, such as: <i>A Tale of Two Brothers</i>, <i>An Orcish Incursion</i>, or <i>The South Guard</i>. Konrad, Li’sar and Delfador are characters from <i>Heir to the Throne</i>. You can also refer to the in-game help browser if you ever need to refresh your memory on gameplay mechanics."
```
Comment (pydsigner) [inline data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg]: While we're in here, it would not hurt to remove that extraneous colon.

**Comment by sigurdfdragon** [comment]
Updated the PR
Changed the AToTB hint to appear on easy and be more general.
Removed the colon in the T S2 commit

**Pull request merged**